.github/workflows/check-ir-stable-names.yml

```diff
@@ -0,0 +1,47 @@
+name: Check the IR stable name table
+
+on:
```

Comment:
can we combine this with other existing workflows to save the time spawning up new tasks and doing repetitive checkouts?